### PR TITLE
Article Page Fixes

### PR DESCRIPTION
**Pull request by Drapegnik**
http://dev.wir.by/be/admin/preview/https%3A%2F%2Fwir.fibery.io%2FContent~Marketing%2Ftest-DEV-Articles-132%23Article%2FTEST-DEV-All-Blocks-Article-1

https://wir.fibery.io/Content~Marketing/test-DEV-Articles-132#Article/TEST-DEV-All-Blocks-Article-1

## blockquote support
Screen Shot 2019-10-11 at 21 57 37

## note
Screen Shot 2019-10-12 at 16 53 36
Screen Shot 2019-10-12 at 16 54 09

## full-width image
Screen Shot 2019-10-12 at 16 53 41

## right table
Screen Shot 2019-10-12 at 16 54 29

## poem
Screen Shot 2019-10-12 at 16 55 46

### CODE REVIEW

**Comment by Drapegnik** [comment]
[Article Page: support blockquote](https://trello.com/c/UY67O9k3/431-article-page-support-blockquote)

**Comment by Drapegnik** [comment]
[Article Page](https://trello.com/c/pEsRW7f9/424-article-page)

File: styles/pages/article.scss [hunk 1]
```diff
@@ -270,57 +271,119 @@
...
+  @include screen-desktop {
+    float: right;
+    margin-left: 30px;
+  }
 
-    @include screen-tablet-large {
-      float: right;
-      margin-left: 30px;
-    }
+  .article-ordered {
```
Comment (UladBohdan) [inline styles/pages/article.scss:431]: Does it really have to be nested?

File: styles/variables.scss [hunk 2]
```diff
@@ -28,3 +28,6 @@ $odnoklassniki-color: #ed812b;
 $error: #e21017;
 
 $tag-image-height: 48px;
+
+$block-spacing: 60px;
+$paragraph-spacing: 24px;
```
Comment (UladBohdan) [inline styles/variables.scss]: How about we put these variables into `styles/pages/article.scss`? Can't see why to make them global.

File: styles/pages/article.scss [hunk 3]
```diff
@@ -333,8 +396,79 @@
...
   display: flex;
+  margin: $block-spacing 0;
+
+  &.right-element {
+    margin-bottom: 0;
+    margin-top: 0;
+  }
 
   th {
     border-bottom: 0.25px solid #cccccc;
```
Comment (UladBohdan) [inline styles/pages/article.scss]: Can we extract this color into variable? #cccccc looks confusing to me :)

**Comment by UladBohdan** [comment]
[Article Page: add collection card](https://trello.com/c/URDm2auN/432-article-page-add-collection-card)

**Comment by UladBohdan** [comment]
@Drapegnik please implement the logic for Collection block on Article page.

**Review by UladBohdan — APPROVED** [review]
Let's merge this.

**Pull request merged**